components/bio/bio.scss

```diff
@@ -72,9 +67,33 @@
     }
 
     &.website {
+      @include compositeGlyph("icon-globe");
+
       &::before {
-        content: "\f0ac";
+        margin-right: 6px;
+      }
+
+      a {
+        font-weight: 700;
       }
     }
   }
+
+  .tags-link {
```

Comment:
okay, sounds good.

Reply:
I'm having trouble doing this in jsx because `{tags}` is returning an array of objects (with objects within them). What would be the best way for me to join these links?

Reply:
Alright. To me it makes sense to connect this list of things with commas as they belong to the same category "interests". I'll let you decide on this since English is not my strongest.

Also it's actually fairly simple to join a list of JSX elements with any "connector" of your choice (in this case, `,`)

e.g.,
```jsx
tags = tags.map((tag, i) => {
    let link = `/${type}/${type === `issues` ? Utility.getUriPathFromIssueName(tag) : encodeURIComponent(tag)}`;

    return <div className="d-inline-block">
      <Link className="tags-link" to={link} key={tag}>{tag}</Link>{i < tags.length-1 && ", "}
    </div>;
});
```

Reply:
Okay, if this is the better way to implement I'll update it.